projects/exui-common-lib/src/lib/components/find-person/find-person.component.ts

```diff
@@ -15,9 +15,12 @@ export class FindPersonComponent implements OnInit {
   @Output() public personSelected = new EventEmitter<Person>();
   @Input() public title: string;
   @Input() public boldTitle = 'Find the person';
+  @Input() public subTitle = 'Type the name of the person and select them.';
   @Input() public domain = PersonRole.ALL;
   @Input() public findPersonGroup: FormGroup;
   @Input() public selectedPerson: string;
+  @Input() public submitted?: boolean = true;
```

Comment:
Why is submitted defaulting to true? Shouldn't this default to false, as the form is not submitted yet when it's first instantiated?

Reply:
@ronaldmansveld submitted is set to true to support legacy implementation of the `find-person.component.ts`.